ecosystem/typescript/sdk_v2/src/core/account.ts

```diff
@@ -0,0 +1,163 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import nacl from "tweetnacl";
+import * as bip39 from "@scure/bip39";
```

Comment:
Any reason we don't just import bip39 and do `bip39.blah`? I feel like it'd be more readable, I'd rather we avoid `import * from x`.